### PR TITLE
[Opperf] Add array rearrange operators to opperf

### PR DESCRIPTION
**Pull request by ChaiBapchya**
## Description ##
Added Rearray operators to the existing set of opperf benchmarks

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [x] Added following ops
1. transpose
2. swapaxes
3. flip
4. depth_to_space
5. space_to_depth

## Results ##
GPU - https://gist.github.com/ChaiBapchya/51b6c7c2346959f3f72aa1b702eb002d
CPU - https://gist.github.com/ChaiBapchya/37b27d4803e73e0fa5a8bd6e8da11492

## Comments ##
@sandeep-krishnamurthy @apeforest

### CODE REVIEW

**Comment by karan6181** [comment]
@mxnet-label-bot add [Operator, Performance, pr-awaiting-review]

File: benchmark/opperf/rules/default_params.py [hunk 1]
```diff
@@ -62,6 +62,15 @@
 # NOTE: Data used is DEFAULT_DATA
 DEFAULT_AXIS = [0]
 
+# For rearrange operators
```
Comment (sandeep-krishnamurthy) [inline benchmark/opperf/rules/default_params.py]: Thanks for this comments. Helps for easy maintenance.

**Comment by ChaiBapchya** [comment]
Ya right. Moreover, functions like
`stop_gradient`, `identity`, `flatten`, `broadcast_plus`, `broadcast_minus` also don't have avg_forward_time. Need to probe. Any idea?

File: benchmark/opperf/utils/op_registry_utils.py [hunk 2]
```diff
@@ -262,6 +264,26 @@ def get_all_sorting_searching_operators():
...
 
 
+def get_all_rearrange_operators():
+    """Gets all array rearrange operators registered with MXNet.
+
+    Returns
+    -------
+    {"operator_name": {"has_backward", "nd_op_handle", "params"}}
+    """
+    rearrange_ops = ['transpose','swapaxes','flip','depth_to_space','space_to_depth']
```
Comment (larroy) [inline benchmark/opperf/utils/op_registry_utils.py]: This should be a set if repeatedly testing for in.

Could we refactor all these get_ functions and make some kind of yaml file where we add tags to the list of operators, I think a declarative thing would be much easier to maintain than all this logic which can be factored out just to select some ops based on the names.

Let me know if what I'm suggesting is not clear.

Comment (larroy) [reply benchmark/opperf/utils/op_registry_utils.py]: Thanks for clarifying this over chat. Seems there's slightly different logics in some of these functions.

**Comment by ChaiBapchya** [comment]
Also, found that `sum_axis` shows `avg_time_forward_sum` instead of `avg_time_forward_sum_axis`

All these operators are aliases and are untracked or wrongly parsed from the profiler output.
Addressing in upcoming commit.

**Pull request merged**